jumpscale/sals/zos/crypto.py

```diff
@@ -1,11 +1,15 @@
 import binascii
+from hashlib import blake2b
 from typing import Union
 
-from nacl.public import SealedBox
+from jumpscale.core.identity import Identity
+from jumpscale.loader import j
+from nacl.bindings import crypto_scalarmult
+from nacl.secret import SecretBox
 from nacl.signing import VerifyKey
 
 
-def encrypt_for_node(public_key: str, payload: Union[str, bytes]) -> str:
+def encrypt_for_node(identity: Identity, public_key: str, payload: Union[str, bytes]) -> str:
```

Comment:
Can we please update the description, because we don't use only the node public key anymore correct?